go/conversation/tests.py

```diff
@@ -839,6 +839,41 @@ def assert_messages(count):
         make_stored_msgs({'sensitive': False})
         assert_messages(2)
 
+    def test_message_list_outbound_status_pending(self):
+        conv = self.user_helper.create_conversation(u'dummy', started=True)
+        self.msg_helper.make_stored_outbound(conv, "hi")
+
+        r_out = self.client.get(
+            self.get_view_url(conv, 'message_list'),
+            {'direction': 'outbound'})
+        self.assertContains(r_out, "<td>pending", html=True)
+        self.assertNotContains(r_out, "<td>sent", html=True)
+        self.assertNotContains(r_out, "<td>failed", html=True)
+
+    def test_message_list_outbound_status_failed(self):
+        conv = self.user_helper.create_conversation(u'dummy', started=True)
+        msg = self.msg_helper.make_stored_outbound(conv, "hi")
+        self.msg_helper.make_stored_nack(conv, msg, nack_reason="no spoons")
+
+        r_out = self.client.get(
+            self.get_view_url(conv, 'message_list'),
+            {'direction': 'outbound'})
+        self.assertContains(r_out, "<td>failed: no spoons", html=True)
```

Comment:
The message and status columns don't have widths specified, so they'll grow or shrink depending on the content. The message list page probably needs a more thorough UX overhaul, but that's out of scope for this PR.